### PR TITLE
Refactor repo

### PR DESCRIPTION
**Pull request by kmagida110**
## Background
This moves files and folders around to create a folder structure where each folder corresponds to related tables in the final database. A build_db script is also included that calls all the relevant functions and builds a final set of CSVs to load to the database.

## Validation Plan
Check output files to ensure they are correct and potentially run `build_db.py` locally.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,5 +10,6 @@ venv/
 src/data_integration/census_data/data/
```
Comment (imcurtis) [inline .gitignore:10]: Is this to keep the src folder separate from the rest of this refactor?

File: README.md [hunk 2]
```diff
@@ -37,17 +31,27 @@ When the database is initially built it needs to be set up with the following st
...
 
 - July 2020 data
   - One time collection of data as of Feb. 2020
   - Stored in bucket referenced in [data retention policy.](https://docs.google.com/document/d/1fBBjWPdC9w8YUlCT47s9-G9jzy0vOQ9ejONviXkkCxI/edit#heading=h.3aiijg3fhho3)
   - Copy sheet `ECE Feb20 Data Collect_All_e` and paste as tab separated CSV (this should be the default) into `src/data_integration/july_2020/data/ece_feb_20_data_collection.csv`.
   - To clean data: go to `src/data_integration/july_2020/` and run `python3 clean_data.py` or `python clean_data.py` depending on your machine's binary for Python 3.
+- Demand Estimation
+  - Contains two estimates of demand created by Skylight:
+    - Number of children under 6 who are under 3.44X of the poverty line which is roughly 75% of the state median income.
+    - Number of children under 6 in families who are under 200% of poverty with 2 working parents.
```
Comment (imcurtis) [inline README.md:51]: I believe this estimate also included families who are under 200% of poverty level with at least 1 working parent also

File: .gitignore (hunk 1 above)
Comment (kmagida110) [reply .gitignore:10]: No, it is to keep intermediate files that aren't used in the final database from being checked into Git.

File: README.md (hunk 2 above)
Comment (kmagida110) [reply README.md:51]: And this is why you are reviewing it, I've updated the README

**Review by imcurtis — APPROVED** [review]

File: src/data_integration/census_data/field_lookup.py [hunk 3]
```diff
@@ -178,29 +179,67 @@ def handle_combination_fields(dat, tables, solo_fields, solo_names, combination_
...
 
-# Let's pull some data
-geo = cd.censusgeo([('state', STATE_CODE), ('county', '*'), ('county subdivision', '*')])
-single_field_mapping = parse_single_field_file(SINGLE_FIELD_FILE)
-df_single = handle_single_fields(geo, single_field_mapping)
-root_names, tables, max_nums, solo_fields, solo_names, combination_fields, combination_names = parse_combination_field_file(COMBINATION_FIELD_FILE)
-df_combination = download_combination_fields(geo, tables, max_nums, solo_fields, combination_fields)
-df_combination = handle_combination_fields(df_combination, tables, solo_fields, solo_names, combination_fields, root_names, combination_names)
-df = df_single.join(df_combination)
-df[df < 0] = nan
```
[the remark this answers is not in the document]
Comment (kmagida110) [reply src/data_integration/census_data/field_lookup.py:191]: Yeah, I just moved it into different functions, I didn't actually delete anything

File: README.md [hunk 4]
```diff
@@ -37,17 +31,28 @@ When the database is initially built it needs to be set up with the following st
...
    - Enable CSV upload and all SQL Lab settings
-1. Use SQL Lab to create uploaded_data schema
+1. Use SQL Lab to create uploaded_data schema and initialize POSTGIS
    - `CREATE SCHEMA uploaded_data;`
-1. Load CSVs as needed 
+   - `CREATE EXTENSION postgis;`
+1. Populate a `config.ini` file in `src/data_integration/connections` using the format in `src/data_integration/connections/config_template.ini`
+1. Run `build_db.py` to create csvs
+1. Load CSVs from `final_data` into tables with the same name as the files.
+1. Load shapefiles using `load_shapefiles_to_db` function in `build_db.py`
```
Comment (jlhogan) [inline README.md:40]: nit this is not an actionable step to someone who hasn't done this before. what does "load" with "function" mean

**Review by jlhogan — APPROVED** [review]
have minimal context here, but changes seem reasonable to me!

**Review by bamader — APPROVED** [review]
The biggest thing I wanted to vet here was that no actual code or artifacts got deleted (because we put good work into those and they're worth keeping around in case we can repurpose their parts later). Doesn't seem like any functions had actual meaning changed or removed, so this is good with me!

**Pull request merged**